src/hearings/models/caseHearing.model.ts

```diff
@@ -13,3 +12,16 @@ export interface CaseHearingModel {
   exuiSectionStatus?: EXUISectionStatusEnum;
   exuiDisplayStatus?: EXUIDisplayStatusEnum;
 }
+export class CaseHearingViewModel implements CaseHearingModel {
```

Comment:
this seems to be duplicated, don't think it's a good way to create another similar model